.changeset/tall-knives-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': patch
+---
+
+- The `<Link />` component now expects the `--source-text-decoration-thickness` CSS variable to be provided to specify the underline thickness on hover.
```

Comment:
I think this makes it sound like users of `Link` need to provide this. how about:
```suggestion
- `<Link />` text has variable underline heights, based on the size of the link text
```
The technique we use to accomplish it is only really an implementation detail. Consumer's don't need to know about it.